client/components/table/index.js

```diff
@@ -179,16 +179,27 @@ class TableCard extends Component {
 			'has-compare': !! compareBy,
 		} );
 
+		const compareButton =
+			selectedRows.length < 2 ? (
+				<Tooltip text={ __( 'Select at least 2 items to compare', 'wc-admin' ) }>
+					<span>
+						<Button isDefault onClick={ this.onCompare } disabled={ true }>
+							{ __( 'Compare', 'wc-admin' ) }
+						</Button>
+					</span>
+				</Tooltip>
+			) : (
+				<Button isDefault onClick={ this.onCompare }>
+					{ __( 'Compare', 'wc-admin' ) }
```

Comment:
This gives us two places to fix button label changes and onClick bindings. Could we DRY this?